src/components/data/test/DataTable-test.js

```diff
@@ -140,20 +175,82 @@ describe('DataTable', () => {
   });
 
   describe('interacting with row data', () => {
-    ['click', 'focus'].forEach(event => {
```

Comment:
The "focus" test here was for testing what happens when the user alters the selection with arrow keys. I have replaced this with a more explicit test of arrow key interaction below.